download.html

```diff
@@ -35,7 +35,7 @@ <h2 id="swanlake">{{ site.data.swanlake-latest.metadata.display-version }} <span
 		</div>
 		<div class="row">
 			<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 cDownloadsHeader">
-				<p> If you are new to Ballerina, download a Ballerina distribution from below based on your operating system and install it.</p>
+				<p>If you are new to Ballerina, download the relevant Ballerina distribution based on your operating system and install it.</p>
```

Comment:
Using "from below" in this context doesn't read correct to me. Can we do without, since the options directly follow the text?